index.html

```diff
@@ -39,17 +38,25 @@
         <div class="dark">
             <header>
 
+                <!--side nav-->
```

Comment:
- [ ] OPTIONAL
Please remove **all** comments in your code hosted on the repository to keep your code cleaner.